### PR TITLE
0.3 update

### PR DESCRIPTION
**Pull request by benfrancis**
- Updated "hero image" on the front page and corresponding Open Graph image
- Updated download links
- Replaced Getting Started presentation with a link to the upcoming Mozilla Hacks blog post
- Linked to the supported hardware page on the wiki

Don't land just yet, need to co-ordinate with the blog posts etc. in time for 9am Pacific Tuesday.

### CODE REVIEW

File: css/site-mobile.css [hunk 1]
```diff
@@ -8,6 +8,7 @@ header #menu-button {
   background-image: url('/images/menu_icon.png');
   background-size: 6.4rem;
   background-repeat: no-repeat;
+  z-index: 2
```
Comment (mrstegeman) [inline css/site-mobile.css]: missing semi-colon

File: gateway/index.html [hunk 2]
```diff
@@ -61,21 +61,19 @@ <h3>Raspberry Pi</h3>
...
+            <a id="download-button" class="button" href="https://github.com/mozilla-iot/gateway/releases/download/0.3.0/gateway-0.3.0.img.zip" onclick="trackOutboundLink('https://github.com/mozilla-iot/gateway/releases/download/0.3.0/gateway-0.3.0.img.zip'); return false;">Download 0.3</a>
           </p>
           <img src="images/sd_card.png" alt="Illustration of an SD card" />
           <p>(Alternatively, if you'd prefer to just try out the software on your PC you can follow the <a href="https://github.com/mozilla-iot/gateway/blob/master/README.md">instructions on GitHub</a> to checkout the code and build it yourself.)</a></p>
-          <p>Once you have successfully flashed the OS onto an SD card you can insert it into your Raspberry Pi and follow the <a href="https://docs.google.com/presentation/d/1urdIH-0vfXYauEW_gshCNyRzAwjcpSx79N_xXWopffI/pub?start=false&loop=false&delayms=3000">Getting Started Guide</a> below:</p>
+          <p><strong>For a more detailed How-To guide and a tour of the gateway's features, see our <a href="how-to-build-your-own-private-smart-home-with-a-raspberry-pi-and-mozillas-things-gateway">blog post</a> on Mozilla Hacks.</strong></p>
```
Comment (mrstegeman) [inline gateway/index.html]: Is this URL just a placeholder for the moment? It's obviously not correct right now.

File: css/site-mobile.css (hunk 1 above)
Comment (benfrancis) [reply css/site-mobile.css]: Done.

File: gateway/index.html (hunk 2 above)
Comment (benfrancis) [reply gateway/index.html]: I *think* this is right now. May have to update it tomorrow.

**Pull request merged**